### PR TITLE
Add port forward script for db connections

### PR DESCRIPTION
**Pull request by davidatkinsuk**
This script replaces the existing remote_db script (when using the ‘-c’ argument to start the JAQY SQL CLI)

### CODE REVIEW

File: script/remote_db_tunnel.sh [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+    echo "Forwarding port. Use ctl-c to close port forwarding"
+    kubectl -n "$NAMESPACE" port-forward $PORT_FORWARD_CONTAINER_NAME $PORT:$PORT
+else
+    # This is ran in the background so we can start the cli
+    # we use trap at the top of this script to kill it on exit
+    kubectl -n "$NAMESPACE" port-forward $PORT_FORWARD_CONTAINER_NAME $PORT:$PORT &
+    port_forward_pid=$!
+    echo "Port forward pid is $port_forward_pid"
+
+    sleep 1
```
Comment (williamfalconeruk) [inline script/remote_db_tunnel.sh:169]: QQ - what's the purpose of the sleep?

Comment (davidatkinsuk) [reply script/remote_db_tunnel.sh:169]: Good question - i think it was returning before the container was ready (presumably by milliseconds) but let me try removing it and seeing if it's stable as it shouldn't really be required. If not stable i'll look at adding a check to make sure the containers ready instead.

Comment (davidatkinsuk) [reply script/remote_db_tunnel.sh]: i've removed the sleep and it seems to work without issue. you already have a 'pod ready' check earlier on in the script, so i'm not sure why i added this!

**Review by williamfalconeruk — APPROVED** [review]
Looking Good!

**Pull request merged**